modules/understanding-upgrade-channels.adoc

```diff
@@ -24,6 +24,7 @@ binary file for a specific version of {product-title} always installs that versi
 * `candidate-{product-version}`
 * `fast-{product-version}`
 * `stable-{product-version}`
+* `eus-4.6` (only available when running 4.6)
```

Comment:
Please do a separate PR that's only for 4.6 to add this instead of mentioning hard coded version here.
Also, please follow OpenShift docs guidelines wrt referencing other products, e.g. {product-title} 4.6.